cmd/hyperkube/kubectl.go

```diff
@@ -19,21 +19,20 @@ package main
 import (
 	"os"
 
-	"k8s.io/kubernetes/cmd/kubectl/app"
+	"k8s.io/kubernetes/pkg/kubectl/cmd"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 )
 
 func NewKubectlServer() *Server {
+	cmd := cmd.NewKubectlCommand(cmdutil.NewFactory(nil), os.Stdin, os.Stdout, os.Stderr)
```

Comment:
So far hyperkube has always collected cmds from ``k8s.io/kubernetes/cmd`` as it adds another layer between ``k8s.io/kubernetes/pkg`` and ``k8s.io/kubernetes/cmd/hyperkube``.  Calling ``cmd.NewKubectlCommand`` right here is against that. Not sure, if it really matters. Still, I see the additional layer as a useful assumption when debugging hyperkube. At the same time the common pattern of defining ``Server.Run`` (as bellow) is to delegate the call to another ``app.Run``.